bin/install_uninstall.ml

```diff
@@ -215,6 +215,33 @@ module File_ops_real (W : Workspace) : File_operations = struct
   let mkdir_p = Path.mkdir_p
 end
 
+module Sections = struct
+  type t =
+    | All
+    | Only of Install.Section.Set.t
+
+  let sections_conv : Install.Section.t list Cmdliner.Arg.converter =
+    let all =
+      Install.Section.all |> Install.Section.Set.to_list
+      |> List.map ~f:(fun section ->
+             (Install.Section.to_string section, section))
+    in
+    Arg.list ~sep:',' (Arg.enum all)
+
+  let term =
+    let doc = "sections that should be installed" in
+    let open Cmdliner.Arg in
+    let+ sections = value & opt sections_conv [] & info [ "sections" ] ~doc in
+    match sections with
+    | [] -> All
```

Comment:
Does that mean that `--sections ""` is interpreted as all sections? That seems a bit odd

Reply:
Yeah, that's true. We can make this an error.